src/ringmesh/basic/aabb.cpp

```diff
@@ -203,7 +206,9 @@ namespace RINGMesh
         {
             return node_index;
         }
-        index_t element_middle, child_left, child_right;
+        index_t element_middle;
+        index_t child_left;
```

Comment:
I think we should use structs to help using these iterators...

Reply:
I agree